core-web/src/main/java/org/silverpeas/core/webapi/notification/user/InboxUserNotificationResource.java

```diff
@@ -86,7 +85,9 @@ protected String getResourceBasePath() {
   @Path("{id}")
   @Produces(MediaType.APPLICATION_JSON)
   public InboxUserNotificationEntity get(@PathParam("id") long id) {
-    return asWebEntity(getMessage(id));
+    return process(() -> asWebEntity(getMessage(getUser().getId(), id)))
+        .lowestAccessRole(null)
```

Comment:
Why do you specify null here as lowestAccessRole as it is the default value of a WebProcess instance?

Reply:
Yes, in fact, it is not necessary to define this value in the case of a GET.
On the other hand, this WEB service is not part of a component instance, and I have certainly thought to indicate precisely here that no role is managed in this context.

Reply:
ok